portfolio/src/main/webapp/script.js

```diff
@@ -30,14 +30,15 @@ function getRandomFact() {
  * Returns comments from the /comments servlet.
  *
  * REQUIRES: none
- * ENSURES: The 'comments-container' div container is replaced with the comments
- *          stored in the /comments servlet as a comma-separated string. If the
+ * ENSURES: The "comments-container" div container is replaced with the comments
+ *          stored in the "/comments" servlet as a comma-separated string. If the
+ *          server returns an error, a nondescript error message is displayed instead.
  */
 function getComments() {
   fetch('/comments')
     .then(handleFetchErrors)
     .then(response => response.json())
-    .then(comments => {
+    .then(commentsInJson => {
       document.getElementById('comments-container').innerText = comments.toString();
```

Comment:
Does this line need to be updated to use "commentsInJson" as well?

Reply:
Oh whoops, thanks for the catch.